ICML-Submission/saliency.py

```diff
@@ -0,0 +1,170 @@
+import tensorflow as tf
+import numpy as np
+from matplotlib.patches import Ellipse
+import matplotlib.pyplot as plt
+import tensorflow.keras.backend as K
+from tensorflow.keras.layers import Input
+from tensorflow.keras.models import Model
+from tensorflow.keras.activations import softmax
+
+
+# change to vis saliency map

```

Comment:
Please rename this to what this is doing, turning a 3 channel image into a grayscale image. something like `convert_to_grayscale`. And snake_case.